src/rrules/performance_patches.jl

```diff
@@ -38,7 +38,9 @@ function rrule!!(
     return zero_fcodual(sum(abs2, x.x)), sum_abs2_pb!!
 end
 
-function generate_hand_written_rrule!!_test_cases(rng_ctor, ::Val{:performance_patches})
+@unstable @unstable function generate_hand_written_rrule!!_test_cases(
```

Comment:
Is there any special meaning of double `@unstable`? 

```suggestion
@unstable function generate_hand_written_rrule!!_test_cases(
```

Reply:
Hm maybe this was from a merge or something.